### ISSUE TITLE
🐞 Bug report: Flexbox Experiment - Container Padding Issue

### ISSUE THREAD

**Issue by perks98** [elementor/elementor#18314]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Please see screenshot below.

![FireShot Capture 057](https://user-images.githubusercontent.com/55829113/160879070-b7add43c-ee25-406e-86d9-3dd743154c88.jpg)

As you will see with boxed width set and padding added, the old sections align properly even with padding, while the new container adds more padding around the sides. I know you've changed it so now for content to be centered you have added padding to make the content area, but please review the CSS below.

```
.e-container {
        padding: var(--padding-top) max(var(--padding-right), calc((100% - var(--content-width)) / 2)) var(--padding-bottom) max(var(--padding-left), calc((100% - var(--content-width)) / 2)) !important
    }
```
What I have added is to the right and left padding using Max(), this then lets the padding work the same way as the sections did. If you made the padding what it is on purpose please let me know why or perhaps if so a toggle button next to the content width dropdown to include or exclude additional left right padding or not. This just makes sure we don't have to have a container within an extra container to fix this.

 

### Steps to reproduce

-

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

-

**Comment by gerardreches** [comment]
Yup, container paddings behaving different than in sections and columns is quite annoying. With sections I would always have 1140px of content even if I added padding, which was convenient. A default padding of 10px for containers inside a container as columns had would be great too, to not have to add it manually every time. It's probably easier to set it to 0 when required than to have to set it to 10px every time.

**Comment by shilo-ey** [comment]
Hey,

This issue has been resolved in Elementor v3.7.0

Feel free to update

Thanks!
